### PR TITLE
Fix broken debugger

### PR DESCRIPTION
**Pull request by anakrish**
1. Update offsets in python plugin gdb_sgx_plugin.py to reflect modifications to _oe_enclave_t struct in host/enclave.h
2. Add comments.
3. Add static asserts to lockdown offsets.

### CODE REVIEW

**Comment by anakrish** [comment]
bors try

File: debugger/pythonExtension/gdb_sgx_plugin.py [hunk 1]
```diff
@@ -17,7 +17,9 @@
 OE_ENCLAVE_HEADER_FORMAT = 'QQQQQ'
 OE_ENCLAVE_MAGIC_VALUE = 0x20dc98463a5ad8b8
 
-OE_ENCLAVE_FLAGS_OFFSET = 0x588
+# The following is the offset of the 'debug' and
+# 'simulate' fields which must lie one after the other.
```
Comment (anitagov) [inline debugger/pythonExtension/gdb_sgx_plugin.py]: Can you add flags somewhere as the debug and simulate are the flags? How about 'simulate' flags?

**Pull request merged**